src/features/event/ui/EventList.tsx

```
@@ -3,6 +3,8 @@ import { useInfiniteScroll } from '../../../shared/hooks/useInfiniteScroll';
 import { getAllEventsInfinite, getCategoryEventsInfinite } from '../../../entities/event/api/event';
 import EventCard from '../../../shared/ui/EventCard';
 import { BaseEvent, CategoryType, TagType } from '../../../shared/types/baseEventType';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
```

ESLint

[error] 'ReactQueryDevtools' is defined but never used.

(@typescript-eslint/no-unused-vars)